scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -161,94 +161,84 @@
 
```

Comment:
I have refactored this big macro by dividing the code into 3 innner macros. It was quite messy before.